package.json

```diff
@@ -4,10 +4,10 @@
   "private": true,
   "description": "Fuel Bridge",
   "scripts": {
-    "format": "turbo run format",
-    "check": "turbo run check",
-    "build": "turbo run build",
-    "test": "sh ./scripts/test.sh",
+    "format": "forc fmt && cargo fmt && turbo run format",
```

Comment:
I can try to make this work, as it does seem valuable to be able to run turbo from any subpackage. The complication is that many of the packages depend on the `/out` dirs to be setup in sibling packages for the paths to work, so since the whole workspace needs to be built anyways I thought it made sense to just run the forc & cargo commands separately at the global level to make sure everything turbo needs is already in place.

Reply:
The advantage of using turbo is that is caches the result, them if you run build again and it was already build and no changes on the files of the package where made it just skips.

Reply:
True, but cargo / forc should be able to handle that themselves even if you rerun the build command several times.